awsiot-optee/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:3.15 as p11
+
+# A bit ahead of 3.15.0, to include support for secure element control (used by SE05X)
+ENV OPTEE_CLIENT_VER "f4f54e5a76641fda22a49f00294771f948cd4c92"
```

Comment:
Didn't change much, but the optee client rev we use in v87 is 3.17 based, which would be hash 9a337049c52495e5e16b4a94decaa3e58fce793e.

Reply:
force pushed a fix.